src/data/vector/vectorFn.ts

```diff
@@ -43,11 +43,24 @@ export const tail: <T extends PLBase>(list: PLVector<T>) => PLVector<T> = (list)
   return plVector(...(list.value.slice(1) as StrictArray<any>))
 }
 
+export const slice: <T extends PLBase>(list: PLVector<T>, start: PLNumber, end: PLNumber) => PLVector<T> = (
```

Comment:
because Pocket Lisp is a functional(-ish) language the target value parameter (in this case the list) must be the last one
```suggestion
export const slice: <T extends PLBase>(start: PLNumber, end: PLNumber, list: PLVector<T>) => PLVector<T> = (
```